configs/native.yaml

```diff
@@ -1,10 +1,14 @@
 plugin_groups:
   - !include "rt_slam_plugins.yaml"
   - plugin_group:
-      - path: ground_truth_slam/
+      # - path: ground_truth_slam/
       - path: gldemo/
       - path: debugview/
-      - path: offload_data/
+      # - path: offload_data/
+      - path: offloaded_vio/device_rx/
+      - path: offloaded_vio/device_tx/
+      # - path: .cache/paths/https%c%s%sgithub.com%sILLIXR%sKimera-VIO.git/
```

Comment:
Remove the `.cache/paths` if they are not needed.

Reply:
Whoops didnt realize i committed native.yaml i can remove that from the PR